extensions/projection/README.md

```diff
@@ -40,43 +40,67 @@ The `proj` prefix is short for "projection", and is not a reference to the PROJ/
 | proj:shape       | \[integer]      | Number of pixels in Y and X directions for the default grid |
 | proj:transform   | \[number]       | The affine transformation coefficients for the default grid  |
 
-**proj:epsg** - A Coordinate Reference System (CRS) is the data reference system (sometimes called a
-'projection') used by the asset data, and can usually be referenced using an [EPSG code](http://epsg.io).
+### proj:epsg
```

Comment:
Sometimes these are h4 (with another h3 "Additional Properties") now and sometimes h3. We should be consistent here.

Reply:
Good catch, will update.